tests/lib/pkgdb.sh

```diff
@@ -104,6 +156,12 @@ distro_update_package_db() {
         ubuntu-*|debian-*)
             quiet apt-get update
             ;;
+        fedora-*)
+            dnf -y -q upgrade
+            ;;
+        opensuse-*)
+            zypper -q update -y
```

Comment:
Funny how similar all the tools are in their CLI interface. I guess apt had a lot of influence here.